src/index.js

```diff
@@ -43,11 +43,15 @@ export default function (CustomElement, opts) {
     static get displayName() {
       return displayName;
     }
+    getNativeElement() {
+      return this._elementNode;
+    }
     componentDidMount() {
+      this._elementNode = ReactDOM.findDOMNode(this);
```

Comment:
Isn't this the same thing as what was happening before?

Reply:
After this change it is exposed. The scoped node element is not accessible outside the funciton before this change.

Reply:
@treshugart Can this be merged? This helps accessing the native element directly

Reply:
I guess what I was wondering is that you'll have access to the component instance. Can't you just call `findDOMNode()` yourself on it?